org.kde.kongress.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "org.kde.kongress",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kongress",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--device=dri",
+        "--share=network",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.Solid.PowerManagement",
+        "--own-name=org.kde.kongressac"
+    ],
+    "modules": [
```

Comment:
You should add a global or per-module `cleanup` for deleting static libraries and development headers.

Reply:
Static libraries and headers cleanup 96fe84c